tests-extension/test/qe/specs/olmv0_defaultoption.go

```diff
@@ -330,6 +331,546 @@ var _ = g.Describe("[sig-operator][Jira:OLM] OLMv0 optional should", func() {
+		exutil.SkipBaselineCaps(oc, "None")
+
+		buildDir := exutil.FixturePath("testdata", "olm")
+		subFile := filepath.Join(buildDir, "olm-subscription.yaml")
+		ogTemplate := filepath.Join(buildDir, "operatorgroup.yaml")
+		prometheusCRTemplate := filepath.Join(buildDir, "prometheus-nodeaffinity.yaml")
+
+		dr := make(olmv0util.DescriberResrouce)
+		itName := g.CurrentSpecReport().FullText()
+		dr.AddIr(itName)
+
+		og := olmv0util.OperatorGroupDescription{
+			Name:      "test-og-54036",
+			Namespace: oc.Namespace(),
+			Template:  ogTemplate,
+		}
+		sub := olmv0util.SubscriptionDescription{
+			SubName:                "sub-54036",
+			Namespace:              oc.Namespace(),
+			CatalogSourceName:      "community-operators",
+			CatalogSourceNamespace: "openshift-marketplace",
+			Channel:                "beta",
+			IpApproval:             "Automatic",
+			OperatorPackage:        "prometheus",
+			SingleNamespace:        true,
+			Template:               subFile,
+		}
+
+		workerNodes, _ := exutil.GetSchedulableLinuxWorkerNodes(oc)
+		firstNode := ""
+		for _, worker := range workerNodes {
+			for _, con := range worker.Status.Conditions {
+				_, ok := worker.Labels["node-role.kubernetes.io/edge"]
+				if con.Type == "Ready" && con.Status == "True" && !ok {
+					firstNode = worker.Name
+				}
+			}
+		}
+		if exutil.IsSNOCluster(oc) || firstNode == "" {
+			g.Skip("SNO cluster - skipping test ...")
+		}
+		if strings.TrimSpace(firstNode) == "" {
+			g.Skip("Skipping because there's no cluster with READY state")
+		}
+
+		g.By("Install the OperatorGroup in a random project")
+		og.CreateWithCheck(oc, itName, dr)
+
+		exists, _ := olmv0util.ClusterPackageExists(oc, sub)
+		if !exists {
+			g.Skip("SKIP:PackageMissing learn does not exist in catalog qe-app-registry")
+		}
+
+		g.By("Install the Prometheus with Automatic approval")
+		sub.Create(oc, itName, dr)
+		olmv0util.NewCheck("expect", exutil.AsAdmin, exutil.WithoutNamespace, exutil.Compare, "Succeeded", exutil.Ok, []string{"csv", sub.InstalledCSV, "-n", oc.Namespace(), "-o=jsonpath={.status.phase}"}).Check(oc)
+
+		g.By("Install the Prometheus CR")
+		err := olmv0util.ApplyResourceFromTemplate(oc, "--ignore-unknown-parameters=true", "-f", prometheusCRTemplate, "-p", "NODE_NAME="+firstNode, "NAMESPACE="+oc.Namespace())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		olmv0util.NewCheck("expect", exutil.AsAdmin, exutil.WithoutNamespace, exutil.Compare, "Available", exutil.Ok, []string{"Prometheus", "example", "-n", oc.Namespace(), "-o=jsonpath={.status.conditions[0].type}"}).Check(oc)
+
+		g.By("Ensure that pod is scaled in the specified node")
+		deployedNode := olmv0util.GetResource(oc, exutil.AsAdmin, exutil.WithoutNamespace, "pods", "prometheus-example-0", "-n", oc.Namespace(), "-o=jsonpath={.spec.nodeName}")
+		o.Expect(firstNode).To(o.Equal(deployedNode))
+	})
+
+	g.It("PolarionID:24850-[OTP]Allow users to edit the deployment of an active CSV", func() {
```

Comment:
@bandrade there is no qe-app-registry catalogsource by default for non-tests-private prow job.
so, you can not use it directly.

if you still want to use learn package, you could refer to case 37442. (if you have other case which use learn package, same comment.)

and more, there is no [Skipped:Disconnected] for this case, so it means it support disconnected env. please refer to https://github.com/openshift/operator-framework-olm/blob/main/tests-extension/test/qe/README.md#disconnected-environment-support-for-migrated-qe-cases to add 'olmv0util.ValidateAccessEnvironment(oc)' at the beginning of the case